models/iaf_psc_exp.h

```diff
@@ -134,6 +139,11 @@ References
 .. [4] Schuecker J, Diesmann M, Helias M (2015). Modulated escape from a
        metastable state driven by colored noise. Physical Review E 92:052119
        DOI: https://doi.org/10.1103/PhysRevE.92.052119
+.. [5] Hanuschkin A, Kunkel S, Helias M, Morrison A and Diesmann M (2010)
```

Comment:
This reference is not needed because it is not mentioned in the text any more.

Reply:
Done!